ui/src/pages/dashboard/ScaleUpDrawer.tsx

```diff
@@ -269,8 +269,8 @@ export default function ScaleUpDrawer({ open, onClose, masterNode }: ScaleUpDraw
                 />
               </FormControl>
               <FormControl fullWidth>
-                <Typography variant="body1" style={{ color: 'red' }}>
-                  ⚠️ Please select a flavor with a disk size of at least 60GB.
+                <Typography variant="body1" style={{ color: 'yellow' }}>
+                  The VM will be created with 60GB disk size.
```

Comment:
Remove this message, document that each agent needs 60GB free disk space either on shared/SAN or ephemeral storage.